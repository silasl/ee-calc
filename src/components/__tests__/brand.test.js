import React from 'react';
import { shallow } from 'enzyme';

import Brand from '../brand';

describe(Brand, () => {
    it('should render correctly', () => {
        expect(shallow(<Brand />)).toMatchSnapshot();
    });
    
    it('should render correctly with a child', () => {
        expect(shallow(<Brand>Foo</Brand>)).toMatchSnapshot();
    });
});
