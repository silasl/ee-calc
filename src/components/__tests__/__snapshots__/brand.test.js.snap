// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brand should render correctly 1`] = `
<styled.div>
  <styled.a
    href="https://www.equalexperts.com/"
    target="_blank"
    title="EE homepage"
  >
    <styled.img
      alt="[=] Equal Experts"
      src="logo.svg"
    />
  </styled.a>
</styled.div>
`;

exports[`Brand should render correctly with a child 1`] = `
<styled.div>
  <styled.a
    href="https://www.equalexperts.com/"
    target="_blank"
    title="EE homepage"
  >
    <styled.img
      alt="[=] Equal Experts"
      src="logo.svg"
    />
  </styled.a>
  Foo
</styled.div>
`;
